### ISSUE TITLE
AVT 1 - Carbon X Radio Button has DAP violation

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#1784]
macOS Mojave version 10.14.2, Chrome version 72.0.3626.96 (Official Build) (64-bit), DAP w/ latest rule set Aug. 2018

Run DAP on Radio Button (See screenshot for example and violations).
<img width="951" alt="screen shot 2019-02-12 at 10 11 06 am" src="https://user-images.githubusercontent.com/21676914/52649874-c2dd9700-2eae-11e9-8ed5-d2b315e44407.png">

Note: The 3rd button is disabled so it should include aria-disabled="true" in the label to fix the contrast issue.

**Comment by snidersd** [comment]
Although the recommended fix will remove the violation it turns out that there is a DAP [GitHub issue 1184](https://github.ibm.com/IBMa/Tools-Rules-HTML/pull/1186) that is causing the violation in error and it should be fixed in the next DAP release expected later this month.
